app/assets/stylesheets/earthworks.css

```diff
@@ -313,3 +313,21 @@ label.toggle-bookmark {
   --bs-link-color-rgb: var(--stanford-digital-blue-dark-rgb);
   text-decoration: underline;
 }
+
+.constraint-value.btn-outline-secondary {
```

Comment:
Secondary is defined as red in the component library.  Can you change the class to be btn-outline-secondary instead?

Reply:
This is in https://github.com/projectblacklight/blacklight/blob/a4ec2d85eeb8d61bd2d462ccdfb5de1c50dba27d/app/components/blacklight/constraint_layout_component.html.erb#L2

Reply:
That is provided by https://github.com/projectblacklight/blacklight/blob/a4ec2d85eeb8d61bd2d462ccdfb5de1c50dba27d/app/components/blacklight/constraint_component.rb#L7
and https://github.com/projectblacklight/blacklight/blob/a4ec2d85eeb8d61bd2d462ccdfb5de1c50dba27d/app/components/blacklight/constraints_component.rb#L26C20-L26C46
Which you can configure in https://github.com/projectblacklight/blacklight/blob/a4ec2d85eeb8d61bd2d462ccdfb5de1c50dba27d/lib/blacklight/configuration.rb#L187